CPAC/pipeline/cpac_pipeline.py

```diff
@@ -1539,8 +1539,7 @@ def prep_workflow(sub_dict, c, run, pipeline_timing_info=None,
 
                     func_preproc = create_func_preproc(
                         skullstrip_tool=skullstrip_tool,
-                        n4_correction=c.n4_correct_mean_EPI,
-                        anatomical_mask_dilation=c.anatomical_mask_dilation,
+                        c=c,
```

Comment:
Is "c" the config? Can we change it to "config" to avoid confusion?
I don't mean the one in cpac_pipeline (can keep that "c" for now), but the one inside the create_func_preproc function, so at least it looks like "config=c,".